### PR TITLE
fix drop shadows appearing in space

### PR DESCRIPTION
**Pull request by Ben10Omintrix**
## About The Pull Request
drop shadows will no longer be appearing in space turfs

## Why It's Good For The Game
fix drop shadows appearing in space

## Changelog
:cl:
fix: fix drop shadows appearing in space
/:cl:

### CODE REVIEW

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Comment by Jacquerel** [comment]
I put off implementing this in this way because I'm worried about the impact of additionally checking the moved turf of every mob on every movement, I was hoping we could come up with some sort of rendering-based solution.
That said, I know absolutely nothing about that area of our codebase or whether that's even possible.

**Comment by optimumtact** [comment]
don't do it based on turfs at all, just use area/entered and have someone manually flag the areas if htey're space or not.

**Comment by Ben10Omintrix** [comment]
> don't do it based on turfs at all, just use area/entered and have someone manually flag the areas if htey're space or not.

could work but one thing that worries me is having floor breaches in station areas (or building floors in space areas) in which case the shadow would incorrectly be hidden/displayed on

File: code/__DEFINES/is_helpers.dm [hunk 1]
```diff
@@ -31,11 +31,18 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 	/turf/open/lava,
 	/turf/open/water,
 	/turf/open/openspace,
-	/turf/open/space/openspace
 	)))
 
 #define isgroundlessturf(A) (is_type_in_typecache(A, GLOB.turfs_without_ground))
 
+GLOB_LIST_INIT(shadowless_turfs, typecacheof(list(
```
Comment (Ghommie) [inline code/__DEFINES/is_helpers.dm]:
```suggestion
GLOBAL_LIST_INIT(shadowless_turfs, typecacheof(list(
```

**Pull request closed without merging**